style.css

```diff
@@ -0,0 +1,66 @@
+*{
+   /* background-image: url(./survey2.jpg); */
+   image-resolution: 50%;
+   
+}
+#main{
+    background-image: url(./survey2.jpg);
+    
+}
+#title{
+    text-align: center;
+    color: red;
+    font-weight: bold;
+}
+.description{
+    text-align: center;
+    font-weight: bold;
+    font-size: 18px;
+}
+#form-div{
+
+    padding-left: 40px;
+
+    padding-right: 40px;
+    padding-top: 40px;
+    padding-bottom: 40px;
```

Comment:
:smiley: Well done applying the padding property. Please consider avoiding repetition in your code.
You can use the code below to replace  the highlighted code.
`padding: 40px 40px;`